bitcoin/src/psbt/mod.rs

```diff
@@ -712,15 +712,17 @@ impl Psbt {
     /// - [`Error::NegativeFee`] if calculated value is negative.
     /// - [`Error::FeeOverflow`] if an integer overflow occurs.
     pub fn fee(&self) -> Result<Amount, Error> {
-        let mut inputs: u64 = 0;
+        let mut inputs = Amount::ZERO;
```

Comment:
This looks better than what I had.  Good idea to use switch to the Amount type here.